File: schemacrawler/src/site/resources/examples/ruby/droptables.rb
puts "WARNING: DROPPING ALL TABLES. RESTART THE DATABASE SERVER TO GET THEM BACK"

statement = $connection.createStatement()
for table in $catalog.tables
	begin
		sql = "DROP " + table.type.toString().upcase + " " + table.fullName
		puts "Executing SQL: " + sql
		statement.executeUpdate(sql)
	rescue Exception => e
		puts e.message
	end
end

File: schemacrawler/src/site/resources/examples/python/droptables.py
print "WARNING: DROPPING ALL TABLES. RESTART THE DATABASE SERVER TO GET THEM BACK"

statement = connection.createStatement()
for table in catalog.tables:
  try:
    sql = "DROP " + table.type.toString() + " " + table.fullName
    print "Executing SQL: " + sql
    statement.executeUpdate(sql)
  except:
    print 'Could not drop ' + table.fullName
    continue